src/Core/Search/Builder/TypedBuilder/FeatureValueFiltersBuilder.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Search\Builder\TypedBuilder;
+
+use PrestaShop\PrestaShop\Core\Language\ContextLanguageProviderInterface;
+use PrestaShop\PrestaShop\Core\Search\Builder\AbstractFiltersBuilder;
+use PrestaShop\PrestaShop\Core\Search\Filters;
+use PrestaShop\PrestaShop\Core\Search\Filters\FeatureValueFilters;
+use Symfony\Component\HttpFoundation\Request;
```

Comment:
```suggestion
use Symfony\Component\HttpFoundation\Request;
```

I see a big problem here.

Reply:
> I see a big problem here.

@FabienPapet, can you elaborate, please? 😅

Is it because a class from the framework is added to the Core namespace? FYI the existing builders already depend on the Request object so I don't think we can block it for this reason. Maybe the builders should actually be part of the `PrestaShopBundle` namespace? Which kind of makes sense as they are used for a ParameterResolver, but then they should be moved in a dedicated PR